### DISCUSSION TITLE
browser.url automatically navigating to https instead of http

### DISCUSSION THREAD

**Discussion by mmalka-tipalti** [webdriverio/webdriverio#13737] [Q&A] [answered]
In my test, I navigate to the HTTP site. For some reason, I can see in debug mode that the first call is to an https site rather than an http site.

The first call - https returns a 404 not found status code. The second call - 404 Temporary Redirect from http site

I tried adding "acceptInsecureCerts: true" but it didn't work

await browser.url("http://blabla.com/demo/iframe?h=moshe.comcskin=JosefBrand");

**Comment by christian-bromann** [comment, marked as answer]
@mmalka-tipalti this sounds to me that the browser is doing this redirect not WebdriverIO. Please check the logs, you should see a navigate command and its parameters, this is what WebdriverIO is sending to the browser.
